File: snippets/php/view_a_contact_s_events.php
require_once('/path/to/MailchimpMarketing/vendor/autoload.php');

$mailchimp = new \MailchimpMarketing\ApiClient();

$mailchimp->setConfig([
    'apiKey' => 'YOUR_API_KEY',
    'server' => 'YOUR_SERVER_PREFIX'
]);

$list_id = "YOUR_LIST_ID";
$email = "urist.mcvankab@example.com";
$subscriber_hash = md5(strtolower($email));

try {
    $response = $mailchimp->lists->getListMemberEvents($list_id, $subscriber_hash);
    echo "We have created {$response->getTotalItems()} events for this user.";
} catch (MailchimpMarketing\ApiException $e) {
    echo $e->getMessage();
}

File: snippets/python/view_a_contact_s_events.py
import hashlib
from mailchimp_marketing import Client

mailchimp = Client()
mailchimp.set_config({
  "api_key": "YOUR_API_KEY",
  "server": "YOUR_SERVER_PREFIX"
})

list_id = 'YOUR_LIST_ID'

subscriber_email = "subscriber@email.com"
subscriber_hash = hashlib.md5(subscriber_email.lower().encode()).hexdigest()

response = mailchimp.lists.get_list_member_events(list_id, subscriber_hash)
total_items = response['total_items']
print(f'We have created {total_items} events for this user.')